package.json

```diff
@@ -37,6 +36,7 @@
     "eslint-plugin-react": "7.35.0",
     "eslint-plugin-react-hooks": "4.6.2",
     "gettext-parser": "8.0.0",
+    "glob": "11.0.0",
```

Comment:
Huh.  Could have sworn I had already done that...

Probably in one of my many unmerged PRs ;)